.github/workflows/at_client_sdk.yaml

```diff
@@ -29,6 +29,11 @@ jobs:
         with:
           sdk: stable
 
+      # Check formatting
+      - name: Check that code in packages is correctly formatted
+        working-directory: packages
+        run: dart format -o none --set-exit-if-changed
```

Comment:
Can we run with --line-length 120 please? The code becomes a lot less readable with the standard line length of 80

If this is all about pub score, maybe instead we should have a step in our publish workflow where we run dart format on the package before we run the publish command

Reply:
I think @murali-shris asked for this for pub scores.

We don't presently use workflows for publishing (we should, for exactly this sort of thing)

Curiously using `--line-length 120` causes a LOT of reformatting with the code as it is:

```
$ dart format . -l 120 -o none --set-exit-if-changed
Changed at_client/example/bin/example.dart
Changed at_client/lib/src/client/at_client_impl.dart
Changed at_client/lib/src/client/at_client_spec.dart
Changed at_client/lib/src/client/local_secondary.dart
Changed at_client/lib/src/client/remote_secondary.dart
Changed at_client/lib/src/client/verb_builder_manager.dart
Changed at_client/lib/src/compaction/at_commit_log_compaction.dart
Changed at_client/lib/src/converters/byte_splitter.dart
Changed at_client/lib/src/converters/encryption/aes_converter.dart
Changed at_client/lib/src/decryption_service/decryption_manager.dart
Changed at_client/lib/src/decryption_service/local_key_decryption.dart
Changed at_client/lib/src/decryption_service/self_key_decryption.dart
Changed at_client/lib/src/decryption_service/shared_key_decryption.dart
Changed at_client/lib/src/encryption_service/abstract_atkey_encryption.dart
Changed at_client/lib/src/encryption_service/encryption_manager.dart
Changed at_client/lib/src/encryption_service/self_key_encryption.dart
Changed at_client/lib/src/encryption_service/shared_key_encryption.dart
Changed at_client/lib/src/encryption_service/signin_public_data.dart
Changed at_client/lib/src/encryption_service/stream_encryption.dart
Changed at_client/lib/src/exception/at_client_exception.dart
Changed at_client/lib/src/key_stream/collection/iterable_key_stream.dart
Changed at_client/lib/src/key_stream/collection/list_key_stream.dart
Changed at_client/lib/src/key_stream/collection/map_key_stream.dart
Changed at_client/lib/src/key_stream/collection/set_key_stream.dart
Changed at_client/lib/src/key_stream/key_stream_impl.dart
Changed at_client/lib/src/key_stream/key_stream_iterable_base.dart
Changed at_client/lib/src/key_stream/key_stream_map_base.dart
Changed at_client/lib/src/key_stream/key_stream_mixin.dart
Changed at_client/lib/src/listener/switch_at_sign_event.dart
Changed at_client/lib/src/manager/at_client_manager.dart
Changed at_client/lib/src/manager/monitor.dart
Changed at_client/lib/src/manager/storage_manager.dart
Changed at_client/lib/src/manager/sync_isolate_manager.dart
Changed at_client/lib/src/manager/sync_manager.dart
Changed at_client/lib/src/preference/at_client_preference.dart
Changed at_client/lib/src/response/at_notification.dart
Changed at_client/lib/src/response/notification_response_parser.dart
Changed at_client/lib/src/service/encryption_service.dart
Changed at_client/lib/src/service/file_transfer_service.dart
Changed at_client/lib/src/service/notification_service.dart
Changed at_client/lib/src/service/notification_service_impl.dart
Changed at_client/lib/src/service/sync/sync_conflict.dart
Changed at_client/lib/src/service/sync_service_impl.dart
Changed at_client/lib/src/stream/file_transfer_object.dart
Changed at_client/lib/src/stream/stream_notification_handler.dart
Changed at_client/lib/src/telemetry/at_client_telemetry.dart
Changed at_client/lib/src/transformer/request_transformer/get_request_transformer.dart
Changed at_client/lib/src/transformer/request_transformer/notify_request_transformer.dart
Changed at_client/lib/src/transformer/request_transformer/put_request_transformer.dart
Changed at_client/lib/src/transformer/response_transformer/get_response_transformer.dart
Changed at_client/lib/src/transformer/response_transformer/notification_response_transformer.dart
Changed at_client/lib/src/util/at_client_util.dart
Changed at_client/lib/src/util/at_client_validation.dart
Changed at_client/lib/src/util/encryption_util.dart
Changed at_client/lib/src/util/sync_util.dart
Changed at_client/test/at_client_impl_test.dart
Changed at_client/test/at_client_util_test.dart
Changed at_client/test/at_key_validation_test.dart
Changed at_client/test/decryption_service_test.dart
Changed at_client/test/default_response_parser_test.dart
Changed at_client/test/encryption_service_test.dart
Changed at_client/test/encryption_util_test.dart
Changed at_client/test/file_encryption_test.dart
Changed at_client/test/key_stream_collection_test.dart
Changed at_client/test/local_secondary_test.dart
Changed at_client/test/monitor_test.dart
Changed at_client/test/notification_service_test.dart
Changed at_client/test/put_request_test.dart
Changed at_client/test/regex_match_util_test.dart
Changed at_client/test/remote_secondary_test.dart
Changed at_client/test/samples/at_client_check_sync.dart
Changed at_client/test/samples/at_client_on_demand_sync.dart
Changed at_client/test/samples/atclient_sync_service.dart
Changed at_client/test/samples/binary_data.dart
Changed at_client/test/samples/file_downloader.dart
Changed at_client/test/samples/file_uploader.dart
Changed at_client/test/samples/local_delete.dart
Changed at_client/test/samples/local_lookup.dart
Changed at_client/test/samples/local_scan.dart
Changed at_client/test/samples/local_scan_with_value.dart
Changed at_client/test/samples/local_update.dart
Changed at_client/test/samples/local_update_with_meta.dart
Changed at_client/test/samples/misc/get_local_commit_log.dart
Changed at_client/test/samples/misc/update_local_commit_log.dart
Changed at_client/test/samples/monitor/monitor_multiple_subscription_regex.dart
Changed at_client/test/samples/monitor/monitor_retry_force_close_socket.dart
Changed at_client/test/samples/monitor/monitor_single_subscription_no_regex.dart
Changed at_client/test/samples/monitor/monitor_single_subscription_regex.dart
Changed at_client/test/samples/remote_llookup.dart
Changed at_client/test/samples/remote_lookup.dart
Changed at_client/test/samples/remote_plookup.dart
Changed at_client/test/samples/remote_scan.dart
Changed at_client/test/samples/remote_stats.dart
Changed at_client/test/samples/remote_update.dart
Changed at_client/test/samples/setup_localkeys.dart
Changed at_client/test/samples/stream_receiver.dart
Changed at_client/test/samples/stream_sender.dart
Changed at_client/test/samples/sync/test_auto_sync_entries_greater_than_threshold.dart
Changed at_client/test/samples/sync/test_auto_sync_entries_less_than_threshold.dart
Changed at_client/test/samples/sync/test_auto_sync_entries_less_than_threshold_remote_ahead.dart
Changed at_client/test/samples/sync/test_auto_sync_switch_atsign.dart
Changed at_client/test/samples/test_util.dart
Changed at_client/test/switch_atsign_test.dart
Changed at_client/test/sync_new_test.dart
Changed at_client/test/sync_service_test.dart
Changed at_client/test/sync_util_test.dart
Changed at_client/test/verb_builder_test.dart
Changed at_client_mobile/example/lib/main.dart
Changed at_client_mobile/example/lib/src/screens/home.dart
Changed at_client_mobile/example/lib/src/screens/login.dart
Changed at_client_mobile/example/lib/src/services/sdk.service.dart
Changed at_client_mobile/lib/src/at_client_auth.dart
Changed at_client_mobile/lib/src/at_client_service.dart
Changed at_client_mobile/lib/src/atsign_key.dart
Changed at_client_mobile/lib/src/keychain_manager.dart
Changed at_client_mobile/test/at_client_service_test.dart
Formatted 159 files (116 changed) in 1.27 seconds.
```